artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/PythonApp_3_Impl_Start_IA-w1/ArtifactTemplate.tosca

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" targetNamespace="http://opentosca.org/artifacttemplates" id="winery-defs-for_artifacttemplates-PythonApp_3_Impl_Start_IA-w1">
+    <ArtifactTemplate xmlns:ToscaBaseTypes="http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes" name="PythonApp_3_Impl_Start_IA-w1" type="ToscaBaseTypes:ScriptArtifact" id="PythonApp_3_Impl_Start_IA-w1">
```

Comment:
I guess I would remove the `Impl` from the artifact name: `PythonApp_3_Start_IA-w1`.
This applies to the other artifacts as well.